syntax/make_commit.sublime-settings

```diff
@@ -0,0 +1,17 @@
+{
+	"always_show_minimap_viewport": false,
+	"auto_indent": false,
+	"detect_indentation": false,
+	"draw_indent_guides": false,
+	"draw_white_space": "None",
+	"font_options": [],
+	"gutter": false,
+	"line_numbers": false,
+	"margin": 20,
+	"match_brackets": false,
+	"match_tags": false,
+	"scroll_past_end": false,
+	"translate_tabs_to_spaces": false,
+	"word_wrap": true,
+	"wrap_width": 100,
```

Comment:
How about word_wrap? Do you think it is a good idea to set a fixed value? Was not sure about it.

Maybe only enabling?

Reply:
I am not sure. I need to think about forcing this on all users without a way to disable it.